pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherSingleActiveConsumer.java

```diff
@@ -537,5 +551,35 @@ public void initializeDispatchRateLimiterIfNeeded(Optional<Policies> policies) {
         return disconnectAllConsumers();
     }
 
+    /**
+     * Get {@link TopicPolicies} for this topic.
+     * @param topicName
+     * @return TopicPolicies is exist else return null.
+     */
+    private TopicPolicies getTopicPolicies(TopicName topicName) {
```

Comment:
Same as the above comment, I think we can move this method to the BrokerService?

Reply:
Thanks for your reply. I have move the getTopicPolicies method to BrokerService, Please take a look again.